src/test/regress/sql/multi_metadata_snapshot.sql

```diff
@@ -59,4 +59,51 @@ SELECT unnest(master_metadata_snapshot());
 UPDATE pg_dist_partition SET partmethod='r' WHERE logicalrelid='non_mx_test_table'::regclass;
 SELECT unnest(master_metadata_snapshot());
 
+-- Test sync_metadata_to_node UDF
+
+-- Ensure that hasmetadata=false for all nodes
+SELECT count(*) FROM pg_dist_node WHERE hasmetadata=true;
+
+-- Run sync_metadata_to_node and check that it marked hasmetadata for that worker
+SELECT sync_metadata_to_node('localhost', :worker_1_port);
+SELECT nodeid, hasmetadata FROM pg_dist_node WHERE nodename='localhost' AND nodeport=:worker_1_port;
+
+-- Check that the metadata has been copied to the worker
+\c - - - :worker_1_port
+SELECT * FROM pg_dist_local_group;
+SELECT * FROM pg_dist_node ORDER BY nodeid;
+SELECT * FROM pg_dist_partition ORDER BY logicalrelid;
```

Comment:
I added `\d tablename` here. Is there a way to limi `\d` to distributed tables only?